### PR TITLE
Add many answer for question

### PR DESCRIPTION
**Pull request by conh-2027**

### CODE REVIEW

**Comment by conh-2027** [comment]
ready

File: app/views/admin/questions/_form.html.erb [hunk 1]
```diff
@@ -12,16 +12,9 @@
...
-      <%= build.label :content, t(".answer") %>
-      <div class="row">
-        <div class="col-8">
-          <%= build.text_field :content, class: "form-control" %>
-        </div>
-        <div class="col-4">
-          <%= build.check_box :is_correct %>
-        </div>
-      </div>
+      <%= render 'answer_fields', f: build %>
```
Comment (hungct-1121) [inline app/views/admin/questions/_form.html.erb]: nháy đôi nhé

**Review by hungct-1121 — COMMENTED** [review]
last comment

**Comment by conh-2027** [comment]
ready

**Review by hungct-1121 — COMMENTED** [review]
(merged)

**Pull request merged**